dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -194,13 +194,8 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 		"integration/docker/conf/alluxio-site.properties.template",
 		"integration/docker/conf/alluxio-env.sh.template",
 		"integration/fuse/bin/alluxio-fuse",
-		"integration/kubernetes/alluxio-configmap.yaml.template",
 		"integration/kubernetes/alluxio-fuse.yaml.template",
 		"integration/kubernetes/alluxio-fuse-client.yaml.template",
-		"integration/kubernetes/alluxio-master-journal-pv.yaml.template",
-		"integration/kubernetes/alluxio-master-journal-pvc.yaml.template",
-		"integration/kubernetes/alluxio-master-service.yaml.template",
-		"integration/kubernetes/alluxio-master-statefulset.yaml.template",
```

Comment:
surprised me that the `worker-daemonset.yaml.template` wasn't in all along... that's a mistake